### ISSUE TITLE
Item authorization page is not usable by normal users

### ISSUE THREAD

**Issue by abollini** [DSpace/dspace-angular#1583]
**Describe the bug**
The item authorization page show a long list of authorization tables related all the objects inside an item. The only information about the contained object are
- his type (Bundle/Bitstream)
- his uuid
The name of the object is absolutely required but missing here.

If the items contains tons of bitstreams only the first 20 bitstreams for each bundle are included without pagination option moreover only objects that have at least one policies are presented so there is no way to restore a policy for a bundle/bitstream once that all the policies have been deleted.

**To Reproduce**
1. create an item with at least one file
2. visit the edit item authorization tab
3. try to guess which authorization table represent the uploaded file and which one the license.txt
4. remove the policy associated with a bitstream, there is not longer a way to add it back
5. take a look to the item https://demo7.dspace.org/entities/publication/d4645ad1-c8df-4133-ab82-bd7b50268a2e that has 100 files for issue related to the pagination

**Expected behavior**
Name should be added to table header for each contained object. UUID can stay, I suggest to remove it from the item authorization table header as it can be already guess from the URL or retrieved in the status tab. The uuid can stay together with the name for bundle and bitstream as they can be useful for experienced adminstrators.
The bundle / bitstreams relation should be more clear. Ideally different sub-tabs for each bundle should be used. If bundle are more than 6 a last "other bundle tab" should be included with all the residual bundle are presented with their bitstreams in a single paginated list. This scenario is very unlikely to happen but it is formally possible as we have 8 lives bundle in our current version ORIGINAL, LICENSE, TEXT, PREVIEW, THUMBNAIL, OtherContent (used by IIIF), DISPLAY, PRESERVATION (used by the coverpage) and a couple of "deprecated ones" CC LICENSE, METADATA.

**Comment by tdonohue** [comment]
Agreed, this sounds like a major usability issue. Pulling onto the 7.3 board.
